building/build-debs/homeworld-admin-tools/resources/__init__.py

```diff
@@ -2,9 +2,11 @@
 
 
 def get_resource(name: str) -> bytes:
-    b = pkgutil.get_data(__package__, name)
+    try:
```

Comment:
Catches error when file not found. Per https://docs.python.org/3.5/library/pkgutil.html.
Should this go in a separate commit, since it is not directly related to spire version?

Reply:
Because it's a change involved in implementing `spire version`, it makes sense to include it in the same PR. You can choose whether it's cleaner in a separate commit within the PR -- whatever seems right.

Reply:
Because it's a change involved in implementing spire version, it makes sense to include it in the same PR. You can choose whether it's cleaner in a separate commit within the PR -- whatever seems right.

Reply:
Moved to separate commit.